charts/vector/values.yaml

```diff
@@ -248,23 +251,24 @@ ingress:
   #        port:
   #          name: ""
   #          number: ""
-  # ingress.tls -- Configure TLS for the Ingress
+  # ingress.tls -- Configure TLS for the Ingress.
   tls: []
   #  - secretName: chart-example-tls
   #    hosts:
   #      - chart-example.local
 
-# existingConfigMaps -- List of existing ConfigMaps for Vector's configuration instead of creating a new one, if used requires dataDir to be set.
-# Additionally, containerPorts, service.ports, and serviceHeadless.ports should be specified based on your supplied configuration
-# If set, this parameter takes precedence over customConfig and the chart's default configs
+# existingConfigMaps -- List of existing ConfigMaps for Vector's configuration instead of creating a new one, if used
+# requires dataDir to be set. Additionally, containerPorts, service.ports, and serviceHeadless.ports should be specified
```

Comment:
```suggestion
# existingConfigMaps -- List of existing ConfigMaps for Vector's configuration instead of creating a new one. Requires
# dataDir to be set. Additionally, containerPorts, service.ports, and serviceHeadless.ports should be specified
```

Reply:
Changed in c725ba6